### PR TITLE
Fix wording in configure-upgrade-etcd.md

### PR DESCRIPTION
**Pull request by eaudetcobello**
A few changes throughout the file to improve wording.

### CODE REVIEW

**Comment by stmcginnis** [comment]
/lgtm

This is a very trivial (but good!) update. Can you take a look through the rest of the file and see if there are any other typos or errors you can fix while doing this?

**Comment by eaudetcobello** [comment]
> /lgtm
> 
> This is a very trivial (but good!) update. Can you take a look through the rest of the file and see if there are any other typos or errors you can fix while doing this?

Sure thing.

**Comment by eaudetcobello** [comment]
> /lgtm
> 
> This is a very trivial (but good!) update. Can you take a look through the rest of the file and see if there are any other typos or errors you can fix while doing this?

Made a few more changes. I removed a link to a closed issue from 2017 here: https://github.com/kubernetes/website/pull/45307/files#diff-7b073284f8f56ad5c59f740e9b3aeaed93c9fe7d5dc188b80bbc74ab55e3a4f0L163

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md [hunk 1]
```diff
@@ -160,9 +160,7 @@ flags `--etcd-certfile=k8sclient.cert`, `--etcd-keyfile=k8sclient.key` and
 `--etcd-cafile=ca.cert`.
 
 {{< note >}}
-etcd authentication is not currently supported by Kubernetes. For more
-information, see the related issue
-[Support Basic Auth for Etcd v2](https://github.com/kubernetes/kubernetes/issues/23398).
+etcd authentication is not planned for Kubernetes.
```
Comment (stmcginnis) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:163]: Personally I think it's fine to remove the link, but will be good to see what others think, if this is good background context to keep in place.

**Review by stmcginnis — COMMENTED** [review]
/lgtm

Thanks, there are some really nice improvements here.

**Comment by stmcginnis** [comment]
Oh, sorry, one more thing. Could you squash your commits? Something like:

```sh
git rebase -i HEAD~4
git push -f origin patch-1
```

**Comment by eaudetcobello** [comment]
> Oh, sorry, one more thing. Could you squash your commits? Something like:
> 
> ```shell
> git rebase -i HEAD~4
> git push -f origin patch-1
> ```

Done

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md (hunk 1 above)
Comment (eaudetcobello) [reply content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:163]: Maybe we should keep it. It atleast tells people "Oh, it was request but there's been a _decision_ not to support it."

Comment (sftim) [reply content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:163]: We [avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future)

How about a follow up PR to rephrase?

**Review by sftim — COMMENTED** [review]
/approve

even with a caveat about the etcd authentication detail

File: content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/configure-upgrade-etcd.md:1]: If revising this page further, note that we [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we) in end user documentation.

**Pull request merged**